lintly/parsers.py

```diff
@@ -527,6 +527,54 @@ def parse_violations(self, output):
         return violations
 
 
+class TfsecParser(BaseLintParser):
+    """
+    Tfsec JSON format
+    {
+        "results": [
+            {
+                "rule_id": "AWS025",
+                "rule_description": "API Gateway domain name uses outdated SSL/TLS protocols.",
+                "rule_provider": "aws",
+                "link": "See https://tfsec.dev/docs/aws/AWS025/ for more information.",
+                "location": {
+                    "filename": "/Users/vnandusekar/git/Lintly/test.tf",
```

Comment:
Redact personal info please! (Don't forget to rebase, so that this doesn't stick around *too* long in the git commit histories either.)

Reply:
:) we should add a precommit hook to look for our usernames or something

Reply:
😔 I always forgot about this. If it is possible to add precommit hook or additional check I would love to add that.